docs/docs/features/state-file.md

```diff
@@ -5,52 +5,89 @@ icon: material/file
 
 ## Introduction
 
-The state file is the serialized state of a recce instance. There are two scenarios where we need to export the state file:
+The state file represents the serialized state of a Recce instance. It is a JSON-formatted file containing the following information:
 
-- [PR Review](#pr-review): We can include the state file in the PR Review Comment. If the reviewer wants to further connect to the PR environment, they can use this file to see the final results and perform deeper audits.
-- [Development](#development): During development, we often need to save the state and even switch between different branches for development.
+- **Checks**: Data from the checks added to the checklist on the Checklist page.
+- **Runs**: Each diff execution in Recce corresponds to a run, similar to a query in a data warehouse. Typically, a single run submits a series of queries to the warehouse and retrieves the final results.
```

Comment:
Dave and I have a discussion on this. We should call it "Checks" before it is added to Checklist.

Reply:
Do you have preferred wording? How about

- **Checks**:Data related to checks on the **Checklist** page.

Reply:
ok. here is to follow the data structure in the state file json, so we need to keep run and check...etc. as the state file.